nasa/run.sh

```diff
@@ -14,16 +14,17 @@ COMPUTE=$5
 
 # Get path to this run.sh script
 basedir=$( cd "$(dirname "$0")" ; pwd -P )
+REPO_ROOT_PATH=$(dirname ${basedir})
 
 # Per NASA MAAP DPS convention, all outputs MUST be placed
 # by the algorithm into a directory called "output".
 # Once the DPS job finishes, MAAP will copy everything from "output"
 # to a directory in my-public-bucket. Everything else on the instance
 # will be destroyed.
-mkdir -p output
+mkdir -p ${REPO_ROOT_PATH}/output
```

Comment:
Input and output paths are relative to the working directory. 
So use something like 
```
mkdir -p ${PWD}/output
```

Reply:
Ah, is this documented anywhere? I looked in the official MAAP documentation, but could not find the method for assigning the location of the output directory. Then I looked at the gdal Hello World example, and it simply used `output` ([link](https://repo.maap-project.org/sshah/gdal-hello-world/-/blob/main/run.sh)).

To confirm -- does this mean that the best practice for any location of the `run.sh` script is to set the output directory to `${PWD}/output`?

Related -- Is there a way to please note in the documentation the convention of how/where to place the `output` dir?

Thank you!

Reply:
Hi @sujen1412 , I just tested and had a successful DPS run using simply `mkdir -p output`.

Should we leave it simple here, or add in `${PWD}/` ?

Thank you again!

Reply:
`${PWD}/` is optional but makes it explicit.